builder.go

```diff
@@ -16,6 +16,7 @@ type (
 		// builder.AddField("SomeFloatField", 0.0, `json:"boolean" validate:"gte=10"`)
 		//
 		AddField(name string, typ interface{}, tag string) Builder
+		addField(name string, pkg string, typ interface{}, tag string, anonymous bool) Builder
```

Comment:
Probably it doesn't need to be part of interface, as it's private method, anyway :)

Reply:
Fixed, please take a review